File: math3.py
number = int(input())
count = 0 # 개수를 세는 변수
sum = 0 # 총합을 구하는 변수
list = []

# 약수의 개수, 총합 구하기
for i in range(1,number + 1):
    if(number % i == 0):
        list.append(i)
        count+=1
        sum+=i
print(number,"의 약수의 개수:",count,"|","약수의 총합:",sum)

File: Math_3.java
import java.util.ArrayList;

public class math3 {
	public static void main(String[] args) {
      //4번 프린트 문제 1번
	  	int num = 36;
	  	int count = 0;
  		int sum = 0;
   		ArrayList list = new ArrayList();
  		for(int i = 1; i <= num; i++) {
		  	if(num % i == 0) {
    				list.add(i);
	  	  	}
	  	}
	  	for(int i = 1; i <= list.size(); i++) {
	  		count++;
	  	}
	  	for(int i = 0; i <= list.size() - 1; i++) {
	  		sum += (int)list.get(i);
	  	}
	  	System.out.println(num + "의 약수의 개수 : " + count);
		  System.out.println(num + "의 약수의 총합 : " + sum);
	    }
    }
